snap/snapcraft.yaml

```diff
@@ -78,14 +78,25 @@ parts:
         cd $SNAPCRAFT_PART_SRC
         python3 tools/strip_hashes.py letsencrypt-auto-source/pieces/dependency-requirements.txt | grep -v python-augeas > snap-constraints.txt
         snapcraftctl set-version `git describe|sed s/^v//`
-
   wrappers:
     plugin: dump
     source: .
     stage: [certbot.wrapper]
+  shared-version:
+    plugin: dump
+    source: .
+    stage: [certbot-version/__init__.py]
```

Comment:
I decided to do it this way because the file basically just contains the version number anyway. I agree that your version is more stable and better. Agreeing to stage just the version number in a specified location is an interface that I'm more comfortable with than assuming that this file will be there in the future. I'll make that change.

Reply:
ooh, if we want to be extra fancy, we could stick it in a json file..... that might be overengineered though, I think I'll just stick it in a file called version or whatever for now

Reply:
Haha nice. I had the same thought and same conclusion yesterday.

I think we could always add that later if there's a need and even in a backwards compatible way if necessary by keeping the version file around.